packages/select/src/ListMenu.tsx

```diff
@@ -98,7 +91,8 @@ const ListMenu = React.forwardRef<HTMLUListElement, ListMenuProps>(
             css`
               font-family: ${fontFamilies.default};
               font-size: ${sizeSet.option.text}px;
-              max-height: ${maxHeight}px;
+              min-height: ${sizeSet.height}px;
+              max-height: ${MAX_MENU_HEIGHT}px;
```

Comment:
All that time wrapping my head around how the Select worked, and this is the solution